pages/models/generations/GENTPJ/index.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: page
+title: GENTPF/GENTPJ Synchronous Machine with Saturation
+tags: ["#196","Synchronous machine", "Rotor", "Stator", "PowerWorld", "Opensource", "GENROU", "GENSAL", "GENTPF", "GENTPJ", "Saturation", ]
+date: 05/08/2024 
+last-updated: 07/08/2024
+id: #196
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+## Context
+
+Saturation modelling in synchronous machines is key to perform transient stability studies. Starting from the dynamic model of the Synchronous Machine, [GENSAL/GENROU without saturation](../GENSAL/) [[1]](#1), some existing models that incorporate the saturation of the poles have been developed. There are some models that directly incorporate the saturation with some additive corrections in the dynamic equation, such as the GENROU/GENSAL with saturation, but they are somewhat obsolete [[2]](#2), and mismatching between the measured field current and voltage measurements with the values predicted have been found. The GENTPF/GENTPJ models [[3]](#3) aimed to apply the saturation effects to the parameters of the machine directly using a scaling factor.
+
+## Model use, assumptions, validity domain and limitations
+
+The model is used to perform transient stability analysis of synchronous machines, providing a more accurate representation of the physical machine as it considers that there exist saturation in the iron of the machine, and it is applied to all the inductive terms simultaneously, differently to what the GENROU/GENSAL models with additive saturation consider. Following a recommendation note from the North-America Electrical Reliability Corporation (NERC) [[4]](4), the GENTPJ is overall recommended for all the synchronous machines as it recognizes the effect of the stator current in saturation. The GENROU, GENSAL and GENTPF (the later being exactly identical to the GENTPJ except an additive term in the saturation function) may induce significant error specially in simulations where reactive power support is considered.
```

Comment:
(NERC) [[4]](4) - link is not working 
![image](https://github.com/user-attachments/assets/f767de37-4ef3-4377-8206-0037db4b3d7d)

Reply:
corrected